src/commands/mod.rs

```diff
@@ -3332,6 +3392,14 @@ fn cmd_sparse_set(
         .iter()
         .map(|v| workspace_command.parse_file_path(v))
         .try_collect()?;
+    // Determine inputs of `edit` operation now, since `workspace_command` is
+    // inaccessible while the working copy is locked.
```

Comment:
Nom just noticed that `locked_wc` (returned by `workspace_command.start_working_copy_mutation()`) borrowed `workspace_command`, so I figured it wouldn't work. I did test it by calling `workspace_command.repo()` where I used `edit_inputs`, and I do get an error:

```
error[E0502]: cannot borrow `workspace_command` as immutable because it is also borrowed as mutable
    --> src\commands\mod.rs:3426:9
     |
3408 |     let (mut locked_wc, _wc_commit) = workspace_command.start_working_copy_mutation()?;
     |                                       ----------------------------------------------- mutable borrow occurs here
...
3426 |         workspace_command.repo();
     |         ^^^^^^^^^^^^^^^^^^^^^^^^ immutable borrow occurs here
...
3443 | }
     | - mutable borrow might be used here, when `locked_wc` is dropped and runs the `Drop` code for type `LockedWorkingCopy`
```